sample/src/App.tsx

```diff
@@ -112,12 +112,28 @@ function AppWithContext({children}: PropsWithChildren) {
   const shopify = useShopifyCheckoutSheet();
 
   useEffect(() => {
-    const subscription = shopify.addEventListener('pixel', event => {
+    const pixelEventSubscription = shopify.addEventListener('pixel', event => {
       // eslint-disable-next-line no-console
       console.log('[PixelEvent]', event?.name, event);
     });
 
-    return () => subscription?.remove();
+    const checkoutCompletedSubscription = shopify.addEventListener(
+      'completed',
+      event => {
+        // eslint-disable-next-line no-console
```

Comment:
I think we should create an eslint config file to ignore the no-console rule to avoid adding it every time we want to use the console.

Reply:
Yeah fair, this is only the sample app so fine with having comments there. Going to add a rule to the package.json to avoid creating more files

Reply:
Agree, I've removed 'em